quelpa-use-package.el

```diff
@@ -37,17 +37,35 @@
 
 ;;; Code:
 
-(defvar quelpa-use-package-inhibit-loading-quelpa nil
+(defgroup quelpa-use-package nil
+  "Quelpa handler for `use-package'."
+  :prefix "quelpa-use-package"
```

Comment:
It's not harmful to specify the prefix, but it's unnecessary for options specified later in the same file.